Tests/PackageLoadingTests/PDLoadingTests.swift

```diff
@@ -17,7 +17,6 @@ import _InternalTestSupport
 import XCTest
 
 class PackageDescriptionLoadingTests: XCTestCase, ManifestLoaderDelegate {
-    lazy var manifestLoader = ManifestLoader(toolchain: try! UserToolchain.default, delegate: self)
```

Comment:
suggestion: can we instantiate `manifestLoader` in a `setUp(...)` call to ensure each test instantiates/creates its own object?